modules/tensor_mechanics/doc/content/source/functions/InitialStrikeShearStress.md

```diff
@@ -0,0 +1,17 @@
+# InitialStrikeShearStress
+
+!syntax description /Functions/InitialStrikeShearStress
+
+## Overview
+
+Custom function for defining initial shear stress spatial distribution of TPV205-2D.
```

Comment:
so this is example specific documentation under the general module. I dont think  that s what the others had in mind

Reply:
sure, we could discuss this tomorrow, I also think this needs to be placed somewhere else